lib/client/users.js

```diff
@@ -131,3 +131,26 @@ Users.prototype.incremental = function (startTime, cb) {
 Users.prototype.incrementalSample = function (startTime, cb) {
   return this.request('GET', ['incremental', 'users', 'sample', { start_time: startTime }], cb);
 };
+
+// ######################################################## User Tags
+
+
+// ====================================== List Tags of User
+Tickets.prototype.listTags = function (userId, tags, cb) {
```

Comment:
Users?....This is currently distributed... :(

Reply:
Fixed in subsequent [PR](https://github.com/blakmatrix/node-zendesk/pull/358) !  It works though :) 
Also extraneous param.
![Screenshot 2023-08-18 at 9 37 08 AM](https://github.com/blakmatrix/node-zendesk/assets/846177/0753d538-9032-45c7-b106-0483b7f729d5)

Reply:
Thanks a bunch for catching this, You're Awesome!

Reply:
NP!  Keeping the magic alive!  New dist working great!